bigflow/scaffold/scaffold_templates.py

```diff
@@ -117,69 +126,36 @@ def count_words(p, target_method):
 '''
 beam_workflow_template = '''import logging
 
-from apache_beam.io import WriteToText
-
+import apache_beam as beam
```

Comment:
Maybe import just Pipeline instead of whole beam?

Reply:
done